docs/architecture_overview/master04-design_principles.adoc

```diff
@@ -1,7 +1,7 @@
 = Design Principles
 
 The openEHR approach to modelling information, services and domain knowledge is based on a
-number of design principles, described below. The application of these principles lead to a separation
+number of design principles, described below. The application of these principles leads to a separation
```

Comment:
"lead" refers to "the application" and not to "these principles", so I think we need a "s" added to it.